config/database.yml

```diff
@@ -1,7 +1,7 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  host: localhost
+  host: <%= ENV['DB_HOST'] %>
```

Comment:
Mind adding the full `DATABASE` here to match 

https://github.com/artsy/echo/pull/19 & the Gris defaults - https://github.com/artsy/gris/pull/73